src/routes/feedRoute.js

```diff
@@ -13,15 +13,18 @@ const {
   activityScore,
   getWeightValue,
   lastP3Score,
-  updatePostScoreGetstream
+  updatePostScoreGetstream,
+  syncMainFeedBroadPerUser
 } = require('../services');
 const auth = require('../middlewares/dev-auth');
 
 const router = express.Router();
 
 router.get('/refresh-post-view-time', refreshPostViewTime);
 // sync user feed
-router.get('/sync-feed/:userId', syncFeedPerUser);
+router.post('/sync-feed', syncFeedPerUser);
+router.post('/sync-main-feed-broad', syncMainFeedBroadPerUser);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The HTTP method for the `sync-feed` and `sync-main-feed-broad` routes has been changed from `GET` to `POST`. This could potentially break existing clients that are using these endpoints. If this change is intentional, make sure to communicate it to all stakeholders and update any client code that uses these endpoints.

```diff
- router.get('/sync-feed/:userId', syncFeedPerUser);
+ router.post('/sync-feed', syncFeedPerUser);
+ router.post('/sync-main-feed-broad', syncMainFeedBroadPerUser);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->